liquibase-maven-plugin/src/main/java/org/liquibase/maven/plugins/LiquibaseRollbackOneChangeSetMojo.java

```diff
@@ -100,7 +100,8 @@ protected void performLiquibaseTask(Liquibase liquibase) throws LiquibaseExcepti
         boolean hasProLicense = MavenUtils.checkProLicense(liquibaseProLicenseKey, commandName, getLog());
         if (! hasProLicense) {
             throw new LiquibaseException(
-                    "The command 'rollbackOneChangeSet' requires a Liquibase Pro License, available at http://www.liquibase.org/download or sales@liquibase.com.");
+                    "The command 'rollbackOneChangeSet' requires a Liquibase Pro License, available at http://www.liquibase.org/download or sales@liquibase.com." +
+                            "Add liquibase.pro.licenseKey=<yourKey> into your defaults file or use --pro-license-key=<yourKey> before your command in the CLI.");
```

Comment:
This code is just for maven, so the --pro-license-key part doesn't apply here. There is a different way we should be explaining how to set it up in maven